src/main/java/com/iexec/sms/secret/app/ApplicationRuntimeSecretService.java

```diff
@@ -61,6 +61,14 @@ public ApplicationRuntimeSecretService(ApplicationRuntimeSecretRepository applic
         return getSecret(appAddress, secretIndex, false);
     }
 
+    /**
+     * Check whether a secret exists.
+     * @return {@code true} if the secret exists in the database, {@code false} otherwise.
+     */
+    public boolean doesSecretExist(String appAddress, long secretIndex) {
```

Comment:
Maybe
```suggestion
    public boolean isSecretPresent(String appAddress, long secretIndex) {
```
?

Reply:
Not sure but I think I prefer the first one. Do you find it unclear?

Reply:
I agree with Zied

Reply:
Alright, that's fixed! Thanks :)

Reply:
Thanks :+1: